### ISSUE TITLE
Second Trial (Paladins) bug

### ISSUE THREAD

**Issue by bwilkinson23** [Looking4Group/L4G_Core#3417]
The quest mobs were in the ground and when I completed the quest and returned to the quest giver (Master Kelerun Bloodmourn) and clicked to interact, only the page that says "Greetings %n" came up. I relogged--same thing. I abandoned the quest and now it shows the quest-giver as offering a quest but still just says "Greetings %n" when I interact with it.

**Comment by Xadras** [comment]
Did you try to delete your cache?
If you did so and it is still not working please provide the exact quest with a link: http://db.hellfire-tbc.com

**Comment by Saltgurka** [comment]
Seem to be happening to other players as well. Some kind of reset issue?

**Comment by rnglol** [comment]
Here is the quest link: http://db.hellfire-tbc.com/?quest=9686

**Comment by AnonXS** [comment]
Maybe need to kill this npc http://db.hellfire-tbc.com/?npc=17807#starts after each attempt.

**Comment by popefreedom** [comment]
Yeah the bug still is not working. It's May 27th, and I stil cant complete this quest. 

quest link: http://db.hellfire-tbc.com/?quest=9690

**Comment by AnonXS** [comment]
it doesnt seem to be bugged for every paladin, so either its only completable once per restart or you are doing something wrong.

**Comment by popefreedom** [comment]
how do u mean by once per restart? Rolling restart?

**Comment by popefreedom** [comment]
Lol, i really want that blue polearm on my paladin
